web/app.php

```diff
@@ -11,6 +11,15 @@
 // When using the HttpCache, you need to call the method in your front controller instead of relying on the configuration parameter
 //Request::enableHttpMethodParameterOverride();
 $request = Request::createFromGlobals();
+
+// Uniquement quand on est dans le cluster k8s
+if (false !== getenv('KUBERNETES_SERVICE_HOST')) {
+    // On fait confiance à toutes les requêtes qui arrivent,  dans le cas ou on est
```

Comment:
Je pense qu'il pourrait être + simple de juste mettre un lien vers https://symfony.com/doc/current/deployment/proxies.html#but-what-if-the-ip-of-my-reverse-proxy-changes-constantly (https://symfony.com/doc/4.0/deployment/proxies.html#but-what-if-the-ip-of-my-reverse-proxy-changes-constantly si on veut un permalink).

Ou sinon apporter un petit changement à la ponctuation (la phrase est un peu longue là)